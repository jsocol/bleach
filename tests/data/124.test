<div id="51"><embed src="data:text/html;base64,PHNjcmlwdD5hbGVydCgxKTwvc2NyaXB0Pg=="></embed>//["'`-->]]>]</div>
