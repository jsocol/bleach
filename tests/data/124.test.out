<div id="51">//["'`--&gt;]]&gt;]</div>
